test/framework/integration_framework/integration_test_framework.hpp

```diff
@@ -38,6 +41,11 @@ namespace shared_model {
     class Block;
   }
 }  // namespace shared_model
+namespace iroha {
+  namespace network {
+    class MstTransportGrpc;
```

Comment:
It does not seem to be used in this file, is it required here?

Reply:
Thanks, that is odd.